### PR TITLE
using python_version to try issues with 0.25.0rc1

### PR DESCRIPTION
**Pull request by nlopezgi**
context: https://github.com/bazelbuild/rules_k8s/issues/305

Creating PR to discuss issues related to 0.25.0rc1 with stakeholders.
Note this will fail until Bazel 0.25.0 is out

### CODE REVIEW

**Comment by nlopezgi** [comment]
Please dont review yet. Just for discussion for now.

fyi @brandjon

File: WORKSPACE [hunk 1]
```diff
@@ -23,12 +23,12 @@ http_archive(
...
 
-# Mention subpar directly to ensure we get a version dated after 2019-03-07,
-# which included fixes for incompatible change flags added in Bazel 0.23. This
+# Mention subpar directly to ensure we get a version dated after 2019-05-13,
+# which included fixes for incompatible change flags added in Bazel 0.25. This
 # can be removed once other dependencies are updated.
 git_repository(
     name = "subpar",
-    commit = "0356bef3fbbabec5f0e196ecfacdeb6db62d48c0",  # 2019-03-07
+    commit = "9c7b3e73b2c1f2befb3a789194bb46982ea336c8",  # 2019-05-13
```
Comment (brandjon) [inline WORKSPACE]: I just tagged subpar 2.0.0, so you can replace this line with `tag = "2.0.0",`.

**Pull request merged**